### PR TITLE
[GSoC] Added Lomax and Bounded pareto distribution

### PR DESCRIPTION
**Pull request by Smit-create**
Added lomax and bounded pareto distribution(CRV types)
#### Brief description of what is fixed or changed
The PR aims at adding two continuous distributions under `crv_types.py`. For adding this distribution, we just need to add the `pdf`, `set`, and `check` to validate the parameters passed as the member functions of the classes. The `rv` is used to map random variables to these classes.
Therefore, to check the `pdf`, and `cdf` of the added RVs, please verify them from,
[https://en.wikipedia.org/wiki/Pareto_distribution#Bounded_Pareto_distribution](https://en.wikipedia.org/wiki/Pareto_distribution#Bounded_Pareto_distribution),
[https://en.wikipedia.org/wiki/Lomax_distribution](https://en.wikipedia.org/wiki/Lomax_distribution)

#### Release Notes

* stats
   * Added Lomax and Bounded pareto distribution

ping @czgdp1807 @jmig5776

### CODE REVIEW

**Comment by czgdp1807** [comment]
LGTM. It would be good if someone can verify the correctness of PDFs, CDFs.

**Comment by jmig5776** [comment]
Please add a good description what the PR is for and a brief explanation of these methods so that other contributors when go through this PR should get this more readable.
I will take a closer look by tonight.

**Comment by Smit-create** [comment]
@jmig5776 The PR aims at adding two continuous distributions under `crv_types.py`. For adding this distribution, we just need to add the `pdf`, `set` and `check` to validate the parameters passed as the member functions of the classes, while all other computations are automatically handled by sympy in `crv.py`, such as `cdf`, `P` , `E`, `variance`, `skewness`, etc. The `rv` is used to map random variables to these classes.
Therefore, to check the `pdf`, and `cdf` of the added RVs, please verify them from,
[https://en.wikipedia.org/wiki/Pareto_distribution#Bounded_Pareto_distribution](https://en.wikipedia.org/wiki/Pareto_distribution#Bounded_Pareto_distribution),
[https://en.wikipedia.org/wiki/Lomax_distribution](https://en.wikipedia.org/wiki/Lomax_distribution)

**Comment by jmig5776** [comment]
> The PR aims at adding two continuous distributions under crv_types.py. For adding this distribution, we just need to add the pdf, set and check to validate the parameters passed as the member functions of the classes, while all other computations are automatically handled by sympy in crv.py, such as cdf, P , E, variance, skewness, etc. The rv is used to map random variables to these classes.
Therefore, to check the pdf, and cdf of the added RVs, please verify them from,
https://en.wikipedia.org/wiki/Pareto_distribution#Bounded_Pareto_distribution,
https://en.wikipedia.org/wiki/Lomax_distribution

Add this to Brief description of PR.

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -2459,6 +2531,71 @@ def LogNormal(name, mean, std):
...
+    @staticmethod
+    def check(alpha, lamda):
+        _value_check(alpha.is_real, "Shape parameter should be real.")
+        _value_check(lamda.is_real, "Scale parameter should be real.")
+        _value_check(alpha.is_positive, "Shape parameter should be positive.")
+        _value_check(lamda.is_positive, "Scale parameter should be positive.")
+
+    def pdf(self, x):
+        lamba, alpha = self.lamda, self.alpha
+        return (alpha/lamba) * (S.One + x/lamba)**(-alpha-1)
```
Comment (czgdp1807) [inline sympy/stats/crv_types.py:2550]: A minor typo but can be ignored.

**Comment by czgdp1807** [comment]
Now I think the task of adding distributions can be taken up by new contributors as it's more of a mechanical process.

**Comment by Smit-create** [comment]
> Now I think the task of adding distributions can be taken up by new contributors

Yeah!, I was thinking the same, and providing more focus to stochastic processes.

File: sympy/stats/crv_types.py (hunk 1 above)
Comment (jmig5776) [reply sympy/stats/crv_types.py:2550]: @Smit-create Can you correct this typo because I don't think we should compromise with code quality here.

Comment (Smit-create) [reply sympy/stats/crv_types.py:2550]: typo of using `1` intead of `S.One`?

Comment (jmig5776) [reply sympy/stats/crv_types.py:2550]: typo of `lamba`->`lamda`

Comment (Smit-create) [reply sympy/stats/crv_types.py:2550]: Oops, will change that in my next PR for sure. Thanks!!

Comment (jmig5776) [reply sympy/stats/crv_types.py:2550]: Cool!!

**Pull request merged**